juju/juju
# Update user facing messages to reference 'agent binaries' instead of 'tools'

## Please provide the following details to expedite Pull Request review:

----

## Description of change

This is a follow on to https://github.com/juju/juju/pull/7354.  It was mentioned in general
to use 'agent binaries' or the like instead of 'tools'.  Updating user facing output accordingly.

## QA steps

1. Verify visually during juju bootstrap --debug and in logs
2. Ran make check in juju/juju

## Documentation changes

Unknown, it may affect some documentation examples?

## Bug reference
